app/helpers/validation/__init__.py

```diff
@@ -38,3 +41,13 @@ def srs_guesser(geom):
                 sr = epsg
                 break
     return sr
+
+
+def validate_sr(sr):
```

Comment:
I kinda feel that "validate_spatial_reference" would be clearer and easier to read. But if you prefer to keep the method original name, no problem with me.